drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/SummaryPage.java

```diff
@@ -0,0 +1,156 @@
+ */
+
+package org.drools.workbench.screens.guided.dtable.client.wizard.column.pages;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import com.google.gwt.user.client.ui.Widget;
+import org.drools.workbench.screens.guided.dtable.client.resources.i18n.GuidedDecisionTableErraiConstants;
+import org.drools.workbench.screens.guided.dtable.client.wizard.column.pages.common.BaseDecisionTableColumnPage;
+import org.drools.workbench.screens.guided.dtable.client.wizard.column.plugins.commons.BaseDecisionTableColumnPlugin;
+import org.drools.workbench.screens.guided.dtable.client.wizard.column.plugins.commons.DecisionTableColumnPlugin;
+import org.jboss.errai.ioc.client.api.ManagedInstance;
+import org.jboss.errai.ui.client.local.spi.TranslationService;
+import org.uberfire.client.callbacks.Callback;
+import org.uberfire.client.mvp.UberElement;
+
+/**
+ * A summary page for the guided Decision Table Wizard
+ */
+@Dependent
+public class SummaryPage extends BaseDecisionTableColumnPage {
+
+    private Boolean includeAdvanced = Boolean.FALSE;
+
+    private List<DecisionTableColumnPlugin> plugins = new ArrayList<>();
+
+    private ManagedInstance<DecisionTableColumnPlugin> pluginManagedInstance;
+
+    private View view;
+
+    @Inject
+    public SummaryPage(final ManagedInstance<DecisionTableColumnPlugin> pluginManagedInstance,
+                       final View view,
+                       final TranslationService translationService) {
+        super(translationService);
+
+        this.pluginManagedInstance = pluginManagedInstance;
+        this.view = view;
+    }
+
+    @Override
+    public String getTitle() {
+        return translate(GuidedDecisionTableErraiConstants.SummaryPage_NewColumn);
+    }
+
+    @Override
+    public void isComplete(final Callback<Boolean> callback) {
+        callback.callback(true);
+    }
+
+    @Override
+    public void prepareView() {
+        view.init(this);
+
+        setupPluginList();
+    }
+
+    @Override
+    protected UberElement<?> getView() {
+        return view;
+    }
+
+    @PostConstruct
+    public void loadPlugins() {
+        final ArrayList<DecisionTableColumnPlugin> loadedPlugins = new ArrayList<DecisionTableColumnPlugin>() {{
+            pluginManagedInstance.forEach(this::add);
+        }};
+
+        this.plugins = sortByTitle(loadedPlugins);
+    }
+
+    void openPage(final String selectedItemText) {
+        if (selectedItemText.isEmpty() || presenter.isReadOnly()) {
+            return;
+        }
+
+        final DecisionTableColumnPlugin plugin = findPluginByIdentifier(selectedItemText);
+
+        wizard.start(plugin);
+    }
+
+    DecisionTableColumnPlugin findPluginByIdentifier(final String selectedItemText) {
+        for (DecisionTableColumnPlugin plugin : this.pluginManagedInstance) {
```

Comment:
Shouldn't this iterate ```plugins()``` and not ```pluginManagedInstance``` (IDK if what you show is instantiating new instances, which won't be terrible as you're using ```ManagedInstance``` but could possibly be avoided?)